spec/forms/base/resetting_attributes_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.describe Forms::Base, "resetting attributes" do
+  let(:form) do
+    Class.new(described_class) do
+      property :foo, type: String, validates: { presence: true }
+      property :bar, type: String
+
+      reset attributes: %i[ bar ], if_falsey: :foo, enabled_value: "yes"
```

Comment:
Any plans to create an RSpec matcher to test for conformance of specific form instances?